Integrations/integration-ESM-v10.yml

```diff
@@ -677,13 +698,14 @@ script:
                 'Name' : organization['name'],
             })
 
+        context = {'Organizations(val.ID && val.ID == obj.ID)' : fixed_organizations}
         return {
             'ContentsFormat': formats['json'],
             'Type': entryTypes['note'],
             'Contents': fixed_organizations,
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': tblToMd('Case Events:', fixed_organizations, headers),
```

Comment:
fix the title to be "Organizations" instead of "Case Events" (old copy-paste...)